protocol/contracts/v2/safeGuards/SafeForked.sol

```diff
@@ -50,7 +78,6 @@ library SafeForked {
         for (i = 0; i < requiredSignatures; i++) {
             (v, r, s) = signatureSplit(signatures, i);
             if (v == 0) {
-                require(keccak256(data) == dataHash, "GS027");
```

Comment:
This is the only functional change (the extra require was added in v1.4.1 branch, which is not yet released)